packages/ckeditor5-engine/tests/model/utils/deletecontent.js

```diff
@@ -317,6 +368,36 @@ describe( 'DataController utils', () => {
 				expect( mergeSpy.called ).to.be.true;
 			} );
 
+			it( 'uses merge operation if first element is empty (because of content delete) and last is not', () => {
```

Comment:
Maybe using "merges elements if..." - the merge operation is an test/implementation detail.